### PR TITLE
Add Mongotron.app 1.0.0-alpha.4

### PR DESCRIPTION
**Pull request by jsynowiec**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download mongotron` is error-free.
- [x] `brew cask style --fix mongotron` left no offenses.
- [x] `brew cask install mongotron` worked successfully.
- [x] `brew cask uninstall mongotron` worked successfully.

### CODE REVIEW

File: Casks/mongotron.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'mongotron' do
+  version '1.0.0-alpha.4'
+  sha256 'b5f435ada45b3a09b6803074b5e2a50883349f2e6a3aa6dfa5bf7c23927d50a1'
+
+  url "https://github.com/officert/mongotron/releases/download/#{version}/Mongotron-darwin-x64.zip"
```
Comment (adidalal) [inline Casks/mongotron.rb]: Needs a `# github.com/officert/mongotron was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by jsynowiec** [comment]
@adidalal Should I update the PR with such comment? I'm not a part of Homebrew-Cask team and it's stated that

> URL was verified by the Homebrew-Cask team as the one provided by the vendor,

**Comment by adidalal** [comment]
Yes please.

(I'm one of the maintainers and I verified the link)

**Comment by vitorgalvao** [comment]
@jsynowiec To be clear, you can always add the comment. We’re the ones who should only merge after we verified it to be accurate.

**Pull request merged**